internal/security/proxy/config/config.go

```diff
@@ -33,8 +35,7 @@ type ConfigurationStruct struct {
 	Clients       map[string]bootstrapConfig.ClientInfo
 }
 
-type WritableInfo struct {
-	LogLevel       string
+type ProxySetupInfo struct {
 	RequestTimeout int
```

Comment:
Why does this need to be in a sub struct? Just have it add the top level also.

Reply:
fixed.